packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -235,7 +255,7 @@ export async function init({
                 }
 
                 if (flexSettings.pluginScript) {
-                    libs.push(pluginScript);
+                    libs.push(pluginScript as string);
```

Comment:
Why does this need to be casted to string?

Reply:
As discussed due to the usage of `FlexSetting` type, where the pluginScripts property doesn't exist, it resolves to `string | boolean` type. In this case the casting here is needed.